src/08-accessing.js

```diff
@@ -148,7 +165,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`
```

Comment:
Long strings like this with backticks can be spread across multiple lines.  We should knock this long string down to at least two if not more lines.